epictrack-api/src/api/services/work.py

```diff
@@ -280,6 +280,15 @@ def create_work(cls, payload, commit: bool = True):
             db.session.commit()
         return work
 
+    @classmethod
+    def _check_duplicate_title(cls, payload, work_id=None):
+        """Check if the title exists."""
+        project = Project.find_by_id(payload.get('project_id'))
+        work_type: WorkType = WorkType.find_by_id(payload.get('work_type_id'))
+        title = util.generate_title(project.name, work_type.name, payload.get('simple_title'))
```

Comment:
if title is a property now, why are we generating it again manually?

Reply:
i just improved variable name and added a comment as well..plz see if that improves